rpm/scratchbox2.spec

```diff
@@ -1,11 +1,10 @@
-Summary:	Scratchbox2 crosscompiling environment
+Summary:	Crosscompiling environment
 License:	LGPLv2
 URL:		https://git.sailfishos.org/mer-core/scratchbox2
 Name:		scratchbox2
 Version:	2.3.90+git58
 Release:	0
 Source:		%{name}-%{version}.tar.gz
-Prefix:		/usr
 ExclusiveArch:	%{ix86} %{x86_64}
```

Comment:
Fix this while at it, x86_64 macro is only available in rpm 4.19 or newer. So replace %{x86_64} with x86_64 for now.